start_kafka.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
-# Start the docker services.
-docker-compose stop
-docker-compose rm -f -v
-docker-compose up --build -d
-docker ps
+./stop_kafka.sh
```

Comment:
![image](https://github.com/usdot-jpo-ode/jpo-cvdp/assets/62157949/6b76f399-b44a-47eb-915f-69a92a80f92c)

Reply:
It's possible that 'chmod 777' needs to be executed on `stop_kafka.sh` prior to using the `start_kafka.sh` script here.

Reply:
yeah, adding 777 works for me.